ts/src/index.ts

```diff
@@ -71,9 +71,9 @@ export async function initConfig(config: Config): Promise<ProfilerConfig> {
   }
 
   let envSetConfig: Config = {};
-  if (process.env.GCLOUD_PROFILER_CONFIG) {
-    envSetConfig =
-        require(path.resolve(process.env.GCLOUD_PROFILER_CONFIG)) as Config;
+  const val = process.env.GCLOUD_PROFILER_CONFIG;
```

Comment:
Should this change be part of this PR?

Reply:
Yes -- when upgrading to @types/node 9.3.0, there is a type error in this section of code without a change.